### PR TITLE
yay: add -Yc example

### PR DESCRIPTION
**Pull request by flyingcakes85**
Add command to remove orphans

- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/yay.md [hunk 1]
```diff
@@ -28,6 +28,10 @@
 
 `yay -Ss {{keyword}}`
 
+- Remove orphan packages
```
Comment (CleanMachine1) [inline pages/linux/yay.md:31]:
```suggestion
- Remove orphan packages:
```

**Review by marchersimon — APPROVED** [review]
Thanks @flyingcakes85

File: pages/linux/yay.md [hunk 2]
```diff
@@ -28,6 +28,10 @@
 
 `yay -Ss {{keyword}}`
 
+- Remove orphan packages (installed as dependencies but not required by any package):
```
Comment (CleanMachine1) [inline pages/linux/yay.md:31]:
```suggestion
- Remove orphaned packages (installed as dependencies but not required by any package):
```

**Comment by flyingcakes85** [comment]
orphan <-> orphaned

That was some grammar to learn :smile:

**Comment by CleanMachine1** [comment]
Thanks, were in the final stage of review now. This will be merged within 24 hours!

**Review by sbrl — APPROVED** [review]
Thanks, @flyingcakes85!

**Pull request merged**